src/Twig/RichEditorExtension.php

```diff
@@ -80,7 +88,7 @@ public function getFunctions()
      *
      * @return string
      */
-    public function renderRichEditorField(string $content)
+    public function renderRichEditorField(string $content): string
```

Comment:
If we change `getRichEditorElements` to `getElements` we can change here to `renderField`